README.md

```diff
@@ -197,5 +194,14 @@ make docker/Dockerfile.b6
 docker build --build-arg=TARGETOS=linux --build-arg=TARGETARCH=amd64 -f docker/Dockerfile.b6 .
 ```
 
+## World Structure
+
+
+![The World Structure](python/docs/b6-world.png)
+
+Data in the world is organised in identifiable features, which can represent anything from a point to an analysis result, with information encoded via tag key/values pairs - e.g. a path mapping to a list of lat/lngs representing the points that belong to it. The initial loaded world is in a compact format encoding, to enable fast search, but, any change done to the world, be it adding a new feature, or editting an existing one, by e.g. adding a tag to it, or changing its ID, is applied in an overlay layer, that is not compact. 
```

Comment:
Maybe instead of _an overlay layer, that is not compact_, something like _an in memory overlay layer, which is less memory efficient than the compact encoding of the initial world.